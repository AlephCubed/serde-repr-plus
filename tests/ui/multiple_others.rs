use serde_repr_plus::Deserialize_repr_clamp;

#[derive(Deserialize_repr_clamp)]
#[repr(u8)]
enum MultipleOthers {
    #[serde(other)]
    A,
    #[serde(other)]
    B,
}

fn main() {}
