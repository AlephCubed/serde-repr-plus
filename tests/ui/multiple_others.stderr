error: only one variant can be #[serde(other)]
 --> tests/ui/multiple_others.rs:3:10
  |
3 | #[derive(Deserialize_repr_clamp)]
  |          ^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `Deserialize_repr_clamp` (in Nightly builds, run with -Z macro-backtrace for more info)
